pkg/controller/baremetalhost/baremetalhost_controller.go

```diff
@@ -43,11 +45,22 @@ const (
 
 var runInTestMode bool
 var runInDemoMode bool
+var maxConcurrentReconciles int = 1
 
 func init() {
 	flag.BoolVar(&runInTestMode, "test-mode", false, "disable ironic communication")
 	flag.BoolVar(&runInDemoMode, "demo-mode", false,
 		"use the demo provisioner to set host states")
+
+	var err error
+	maxConcurrentReconciles, err = strconv.Atoi(os.Getenv("OPERATOR_CONCURRENCY"))
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "OPERATOR_CONCURRENCY variable not found. Using default: 1")
```

Comment:
maybe we could use log instead of printf here?

Also the message isn't entirely clear because we're combining two possible outcomes, "OPERATOR_CONCURRENCY is either unset or invalid" perhaps.

Reply:
WIll do, thanks. I'll log the message.